revapi-reporter-file-base/src/main/java/org/revapi/reporter/file/AbstractFileReporter.java

```diff
@@ -166,6 +172,7 @@ public void report(@Nonnull Report report) {
         LOG.trace("Received report {}", report);
 
         if (report.getDifferences().isEmpty()) {
```

Comment:
I believe this never happens in "reality", because reporters are only called if there are some differences - https://github.com/revapi/revapi/blob/2664b48f5ebd94a45c8a17ffc628441f77154426/revapi/src/main/java/org/revapi/Revapi.java#L465. Also, be aware that this method is called many times - for each differing element pair.